utils/types/deployment/deployment_test.go

```diff
@@ -137,7 +137,7 @@ func Test_GetConnectionToken(t *testing.T) {
 			namespace:          "free-us-1",
 			expectedIdentifier: "",
 			expectedBool:       false,
-			err:                fmt.Errorf("invalid deployment type: \"badType\""),
+			err:                fmt.Errorf(`invalid deployment type: \"badType\"`),
```

Comment:
Is the escaping of the double quotes needed now that you're using backtickers?

```suggestion
			err:                fmt.Errorf(`invalid deployment type: \"badType\"`),
```